components/style/font_metrics.rs

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use Atom;
+use app_units::Au;
+use euclid::Size2D;
+use std::fmt;
+
+/// Represents the font metrics that style needs from a font to compute the
+/// value of certain CSS units like `ex`.
+#[derive(Debug, PartialEq, Clone)]
```

Comment:
Nope, that's intentional, just in case it becomes bigger, or a `RefPtr` to a gecko struct, or whatnot.